# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: f95a1c5ef0b1
# Full Hash: f95a1c5ef0b1652e911f6ff96c6c38796856e70e
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2021-08-13 09:27:46
# Regressor Bug: 1724141
# File Overlap Count: 6
# Description:
#   Bug 1724141 - Update audioipc to 8bb1a227.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D121826
# ==============================================================================

diff -r c9c0a125e6b2 -r f95a1c5ef0b1 .cargo/config.in
--- a/.cargo/config.in	Thu Aug 12 21:12:32 2021 +0000
+++ b/.cargo/config.in	Thu Aug 12 21:28:03 2021 +0000
@@ -40,7 +40,7 @@
 [source."https://github.com/mozilla/audioipc-2"]
 git = "https://github.com/mozilla/audioipc-2"
 replace-with = "vendored-sources"
-rev = "7537bfadad2e981577eb75e4f13662fc517e1a09"
+rev = "8bb1a227fbaa5677458bcd876162b65307df38c2"
 
 [source."https://github.com/mozilla/application-services"]
 git = "https://github.com/mozilla/application-services"